src/components/common/form/PostForm.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <div>
```

Comment:
```suggestion
```
all the elements are already wrapped in BaseForm, it does not make sense to have an extra div wrapper